python/lsst/pipe/analysis/fakesAnalysis.py

```diff
@@ -1173,4 +1264,102 @@ def fakesCompletenessPlot(inputFakes, inputFakesMatched, processedFakesMatched,
     description = "completenessHist"
     stats = None
     yield pipeBase.Struct(fig=fig, stats=stats, description=description, style="fakes")
-    plt.close()
+
+
+def fakesMagnitudePositionError(inputFakesMatched, processedFakesMatched, plotInfoDict, areaDict,
+                                raFakesCol="raJ2000_deg", decFakesCol="decJ2000_deg", raCatCol="coord_ra_deg",
+                                decCatCol="coord_dec_deg", magCol="base_PsfFlux_mag"):
+    """Makes plots showing the comparison between the input minus the extracted magnitudes and the
+       difference in the input and extracted positions, one showing the position difference locations
+       in R.A. and Dec., one showing position differences against output - input magnitude difference
+       and one showing position errors against input magnitude.
+
+    Parameters
+    ----------
+    inputFakesMatched : `pandas.core.frame.DataFrame`
+        The catalog used to add the fakes originally matched to the processed catalog.
+    processedFakesMatched : `pandas.core.frame.DataFrame`
+        The catalog produced by the stack from the images with fakes in.
+    plotInfoDict : `dict`
```

Comment:
Tell the user what needs to be in the dict: https://developer.lsst.io/python/numpydoc.html#dictionary-types